packages/graphback/src/schema/targetSchemaContext.ts

```diff
@@ -132,37 +132,43 @@ export const buildTargetContext = (input: Type[]) => {
   const inputContext = input.filter((t: Type) => t.name !== 'Query' && t.name !== 'Mutation' && t.name !== 'Subscription')
 
   const relations = []
+  let relation = {}
 
   inputContext.forEach((t: Type) => {
     t.fields.forEach((f: Field) => {
       if(f.isType){
         if(f.directives.OneToOne || !f.isArray) {
-          relations.push({
+          relation = {
             "name": t.name,
             "type": 'Type',
             "relation": `${f.name}: ${f.type}`
-          })
-          relations.push({
+          }
+          if (!relations.includes(relation)) relations.push(relation)
+          relation = {
             "name": f.type,
             "type": 'ID',
-            "relation": `${t.name.toLowerCase()}Id: ID!`
-          })
+            "relation": `${f.name}Id: ID!`
```

Comment:
I think this should be `${f.type.toLowerCase()}: ${f.type}`, this will give you the `user: User` field

Reply:
Ah, I think you are right. It's because `f.type !== f.name` if someone used a different name for the field. Good catch. I was lucky because in my example schema they are the same. I'll make that change.

Reply:
I pushed that change to my branch but am not seeing how to merge it in to this PR. Is that possible?

Reply:
If you pushed it to your branch, it should automatically reflect in the PR.